bookkeeper-server/src/test/java/org/apache/bookkeeper/test/ZooKeeperCluster.java

```diff
@@ -59,7 +59,7 @@ public interface ZooKeeperCluster {
 
     void killCluster() throws Exception;
 
-    void sleepCluster(int time, final TimeUnit timeUnit, final CountDownLatch l)
+    void sleepCluster(int time, TimeUnit timeUnit, CountDownLatch l)
```

Comment:
is it related to puppycrawl version upgrade?

Reply:
yes